keylime/cloud_verifier_tornado.py

```diff
@@ -421,6 +422,24 @@ def post(self):
                     logger.warning("POST returning 400 response. Expected non zero content length.")
                 else:
                     json_body = json.loads(self.request.body)
+
+                    # Handle allowlists/IMA policies
+                    ima_policy_bundle = json.loads(json_body.get("ima_policy_bundle"))
+                    if ima_policy_bundle:
+                        try:
+                            allowlist = ima.unbundle_allowlist(ima_policy_bundle)
```

Comment:
This is still confusingly named. `unbundle_allowlist` returns an allowlist... but not really. You extract the ima_policy from the allowlist but then used it as the key for agent_data's "allowlist". It seems to get things backward. An ima_policy would include an allowlist (along with an exclude list, signatures, etc, right?). So maybe `unbundle_allowlist` should be renamed `unbundle_ima_policy` and should return an ima_policy which you then pass to `process_ima_policy` to get an `allowlist` which you then pass as the `allowlist` key to the `agent_data`.

Or maybe I'm just more confused? This might seem nit-picky, but naming things is important to understand the flow. Something shouldn't be called an allowlist if it's more than an allowlist, etc. And it shouldn't be called an ima_policy if it's actually an allowlist.

Reply:
@mpeters I think the _most accurate_ naming would be if the `allowlist` key in `agent_data` changed to `ima_policy`. The "allowlist" sent over the wire prior to this PR and put into that key was actually closer to an IMA policy - it contained both `allowlist` and `excllist` subkeys. Hence why I named the variable I'm putting into `agent_data` as `ima_policy`. You're right about `unbundle_allowlist`, though. I'll change that.

My understanding is that if we wanted to change how things are named in `agent_data`, we'd need to change the database schema as well as update it where it's accessed. I'm not opposed to that, but IIRc it's not just a simple rename in that case.

Reply:
> @mpeters I think the _most accurate_ naming would be if the `allowlist` key in `agent_data` changed to `ima_policy`. 

You're right and that's probably where my confusion is coming from.

> My understanding is that if we wanted to change how things are named in `agent_data`, we'd need to change the database schema as well as update it where it's accessed. I'm not opposed to that, but IIRC it's not just a simple rename in that case (wouldn't it also involve a migration of some sort?).

We don't need to change that database field with this work. We can/should leave that to the larger IMA policy re-work.

Reply:
@mpeters Sounds good. I should have made that detail clearer. I'll rename the `process_allowlist` function - otherwise, does the rest of the naming make sense given that context?

Reply:
yes